rtris.py

```diff
@@ -4,11 +4,176 @@
 from numpy import add as np_add
 os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
 import pygame,pygame.freetype
+from sys import argv
 
 K_DROP=False
 confpath=os.path.abspath(os.path.expanduser("~/.rtrisconf"))
 
+VERSION="1.0a3"
+COPYRIGHT_YEAR="2019"
+# TODO: do you wanna give out your full name?
+COPYRIGHT_HOLDER="RIEDLER"
+AUTHORS="RIEDLER and Michael Federczuk"
+
+USAGE="usage: %s" % (argv[0])
+HELP="""\
+%s
+    Start Rtris, a Tetris clone written in Python.
+
+    Options:
+      -h, --help      display this summary and exit
+      -V, --version   display version information and exit
+      -d, --debug     print debug information (default)
+      -D, --no-debug  don't print debug information
+
+    Exit Status:
+      (using CommonCodes v1.0.0
+       <https://mfederczuk.github.io/commoncodes/v/1.0.0.html>)
+
+Report bugs at: https://github.com/RiedleroD/Rtris/issues
+Rtris repository: https://github.com/RiedleroD/Rtris""" % (USAGE)
+VERSION_INFO="""\
+Rtris %s
+Copyright (c) %s %s
+License CC BY-SA 4.0: Creative Commons Attribution-ShareAlike 4.0 <http://creativecommons.org/licenses/by-sa/4.0>
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Written by %s.""" % (VERSION, COPYRIGHT_YEAR, COPYRIGHT_HOLDER, AUTHORS)
+
+def eprint(*args, **kwargs):
+	from sys import stderr
```

Comment:
this should get imported at the start of the file, not the start of the function